doc/usage/restructuredtext/directives.rst

```diff
@@ -598,10 +598,23 @@ __ https://pygments.org/docs/lexers
       are removed via :func:`textwrap.dedent()`.  For example::
 
          .. code-block:: ruby
+            :linenos:
             :dedent: 4
 
                 some ruby code
 
+      Using ``0`` as argument does not strip anything and can be used to
```

Comment:
As I commented in the issue, I can agree on this feature as a "secret" feature. So this should not be mentioned in the document. To add a proper feature, we have to consider another way to keep indentation because it's common problem of reST directives, not only this directive.

Reply:
Ok, I'll remove this section.